pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/logging/LogUtil.java

```diff
@@ -1,5 +0,0 @@
-package org.broadinstitute.ddp.logging;
-
```

Comment:
Moved to [here](https://github.com/broadinstitute/ddp-study-server/pull/2854/files#r1570786921) so that DSM and DSS can use a single class for this.